content/en/security/security_inbox.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Security Inbox
+kind: documentation
+further_reading:
+- link: "/security/application_security/"
+  tag: "Documentation"
+  text: "Learn more about Application Security Management"
+- link: "/security/cloud_security_management"
+  tag: "Documentation"
+  text: "Learn more about Cloud Security Management"
+- link: "/security/default_rules/#all"
+  tag: "Documentation"
+  text: "Out-of-the-box Detection Rules"
+---
+
+Security Inbox provides a consolidated, actionable list of your most important security findings. It automatically contextualizes and correlates insights from Datadog security products across vulnerabilities, signals, misconfigurations, and identity risks into a unified, prioritized view of actions to take to strengthen your environment.
+
+{{< img src="security/csm/security_inbox_2.png" alt="The Security Inbox shows prioritized security issues for remediation" width="100%">}}
+
+## Types of findings in Security Inbox
+
+The findings that appear in Security Inbox are generated from Application Security Management (ASM) and Cloud Security Management (CSM). These include the following types of findings:
+
+- [Misconfigurations][1] for [CSM Misconfigurations][2].
+- [Identity risks][1] for [CSM Identity Risks][3].
+- Application library vulnerabilities for [ASM library vulnerabilities][4]. All high and critical application library vulnerabilities on production services under attack appear in the inbox.
+- Application code vulnerabilities for [ASM code-level vulnerabilities][5]. All high and critical application code vulnerabilities appear in the inbox.
+- [Attack Paths][1]. An attack path outlines a series of interconnected misconfigurations, container image, host, and application vulnerabilities that malicious actors could leverage to gain unauthorized access, escalate privileges, or compromise sensitive data in your cloud environment. All attack paths are listed in Security Inbox by default.
+
+Security Inbox also takes the following detected risks into consideration when determining which findings appear in the inbox:
+
+- **Public accessibility**: Publicly exposed resources carry elevated risk, especially if they contain vulnerabilities or misconfigurations. To learn more, see [How Datadog Determines if Resources are Publicly Accessible][6].
+- **Privileged access**: Resources with privileged access carry elevated risk as they grant elevated permissions that can expand the attack surface. To learn more, see [How Datadog Determines if Resources are Publicly Accessible][6].
```

Comment:
probably a copy-paste mistake here

Reply:
This was noted as something we needed to add in the original doc. But you're right, it doesn't make sense to have it here. I'll remove it from the PR.

Reply:
we should probably have a doc page that says "here's how we consider if something is privileged"